javascript/src/URDFClasses.js

```diff
@@ -237,9 +244,35 @@ class URDFJoint extends URDFBase {
             }
 
             case 'floating':
-            case 'planar':
                 // TODO: Support these joint types
                 console.warn(`'${ this.jointType }' joint not yet supported`);
+                break;
+            case 'planar':
+                const posX = values[0];
+                const posY = values[1];
+                const rotZ = values[2];
+
+                if (posX == null && posY == null && rotZ == null) return didUpdate;
```

Comment:
So to be clear we'd be enabling the function to take "null" to mean "no change" to that axis dimension - is that right? In this case I agree it's a bit odd to blindly parse the values. I'm thinking it's cleaner to only parse the non-null values at the beginning of the function which would make this logic more clear.

```js
values = values.map(v => v === null ? null : parseFloat(v));
```

Reply:
That makes sense to me, I can make that change to skip parsing `null` array elements either in this PR or another since it's not necessarily directly related to my code here, whichever is your preference.

Reply:
Whichever works for me. I'm fine with a separate PR